### DISCUSSION TITLE
How to fix the blur?

### DISCUSSION THREAD

**Discussion by txtyash** [qutebrowser/qutebrowser#7987] [Q&A]
I am facing the exact same issue as #1291 and I did not find any satisfactory answer there.
![image](https://github.com/qutebrowser/qutebrowser/assets/85027668/b0c86890-f238-44f8-b426-88a05715333a)

**Comment by txtyash** [comment]
I'm using nixos with gnome.
uname -a
```bash
Linux vivobook 6.1.60 #1-NixOS SMP PREEMPT_DYNAMIC Wed Oct 25 10:03:17 UTC 2023 x86_64 GNU/Linux
```

**Reply by The-Compiler** [reply]
Can you show your full `:version` information?

**Reply by txtyash** [reply]
https://bpa.st/EYAA

**Reply by The-Compiler** [reply]
Does anything change with `QT_QPA_PLATFORM=wayland-egl qutebrowser --temp-basedir`?

**Reply by txtyash** [reply]
IG not
![image](https://github.com/qutebrowser/qutebrowser/assets/85027668/44fade91-93e2-4ed4-b785-91c43a31494b)

**Reply by The-Compiler** [reply]
No idea then. Given that HighDPI screens are quite common and nobody else is seeing this, this might be something specific to your system. But qutebrowser itself doesn't really do much in terms of HighDPI scaling, it's all done by the underlying Qt.
